en/micro-integrator/docs/references/connectors/db-event-inbound-endpoint/db-event-inbound-endpoint-example.md

```diff
@@ -0,0 +1,139 @@
+
+In the system of an enterprise a relational database table is used to store customer information. Customers' information is added by an external system to the database which is not in enterprise's control. As soon as a new customer is inserted, the system need to pick up and process its data. `WSO2 EI` is used here to listen to DB changes and invoke the relevant processes. It can invoke backend APIs or place data into a message bus after required data transformations. However, for simplicity of this example, we will just log the message. You can extend the sample as required using WSO2 EI mediators. 
+
+Following diagram shows the overall solution we are going to build.
+
+<img src="/assets/img/connectors/db-event-diagram.png" title="Overview of DB event inbound EP use case" width="600" alt="Overview of DB event inbound EP use case"/>
+
+
+## Setting up the environment 
+
+First, install [MySQL database](https://www.mysql.com/downloads/) locally. If you have a remote server, please obtain credentials required to connect. Here in this example database credentials are assumed as username=`root` and password=`root`. 
+
+1. Create a database called `test`. Then create a table called `CDC_CUSTOM` under that database using following SQL script. 
+```sql
+CREATE TABLE `test`.`CDC_CUSTOM` (
+  `ID` INT NOT NULL,
+  `NAME` VARCHAR(45) NULL,
+  `ADDRESS` VARCHAR(45) NULL,
+  `AMOUNT` INT NULL,
+  PRIMARY KEY (`ID`));
+```
+
+2. In order to track new records we need an additional column. If you apply this feature to an existing database table, you can alter the table as below. It will add a column of type `TIMESTAMP` which gets automatically updated upon insert or update of a record. 
+```sql
+ALTER TABLE CDC_CUSTOM
+ADD COLUMN UPDATED_AT 
+  TIMESTAMP DEFAULT CURRENT_TIMESTAMP 
+  ON UPDATE CURRENT_TIMESTAMP;
+```
+
+
+## Configure inbound endpoint using WSO2 Integration Studio
+
+1. Download [WSO2 Integration Studio](https://wso2.com/integration/integration-studio/). Create an **ESB Solution Project** as below. 
+<img src="/assets/img/connectors/solution-project.png" title="Creating a new ESB Solution Project" width="800" alt="Creating a new ESB Solution Project" />
+
+2. Right click on **Source** -> **main** -> **synapse-config** -> **inbound-endpoints** and add a new **custom inbound endpoint**.</br> 
+<img src="/assets/img/connectors/db-event-inbound-ep.png" title="Creating DB event inbound endpoint" width="400" alt="Creating DB event inbound endpoint" style="border:1px solid black"/>
+
+3. Click on **Inbound Endpoint** in design view and under `properties` tab, update class name to `org.wso2.carbon.inbound.poll.dbeventlistener.DBEventPollingConsumer`. 
+
+4. Navigate to source view and update it with following config. Please note that you need to update url, username and password as required. 
+```xml
```

Comment:
Tab this code block so that it appears as part of the list.

Reply:
updated.